diagnostics/stc_vert_wave_coupling/doc/stc_vert_wave_coupling.rst

```diff
@@ -0,0 +1,223 @@
+the NH JFM seasons, and SH SON seasons, which are used to plot composite
+maps during extreme heat flux events. 
+
+
+Required programming language and libraries
+-------------------------------------------
+
+This POD requires Python 3, with the following packages:
+
+- numpy
+- scipy
+- xarray
+- pandas
+- matplotlib
+
+
+Required model output variables
+-------------------------------
+
+The following daily mean fields are required:
+
+- Temperature at 50 hPa, ``ta50`` as ``(time,lat,lon)`` (units: K)
+- Meridional wind at 50 hPa, ``va50`` as ``(time,lat,lon)`` (units: m/s)
+- Geopotential Height at 10 hPa, ``zg10`` as ``(time,lat,lon)`` (units: m)
+- Geopotential Height at 500 hPa, ``zg500`` as ``(time,lat,lon)`` (units: m)
+
+
+Scientific background 
+---------------------
+Wave motions in the polar stratosphere are primarily dominated by 
+vertically propagating Rossby waves from the troposphere. Because of 
+so-called "Charney-Drazin filtering", only the largest planetary scale 
+waves are able to propagate into the stratosphere when there are westerly 
+mean winds (Charney & Drazin, 1961; Andrews et al., 1987). As a result, 
+vertical wave coupling between the troposphere and stratosphere follows a 
+distinct seasonal cycle organized around the formation of the westerly 
+stratospheric polar vortex in autumn (when waves can enter the stratosphere), 
+and its breakdown in spring/early summer (when easterly winds prevent propagation).
+
+The propagation characteristics of planetary waves are strongly dependent on 
+the background mean flow, which can influence where/how these waves propagate. 
+In some cases these can lead to events in which waves are reflected from the 
+stratosphere back into the troposphere, which tend to occur most often in late 
+winter in the NH and spring in the SH (Shaw et al., 2010). These reflected 
+waves can directly influence the tropospheric circulation. 
+
+Wave events in the stratosphere are associated with meridional fluxes of heat
+that can be characterized by "eddy heat fluxes" (v'T', where v is the 
+meridional wind, and T is the temperature, and primes denote deviations from
+the zonal mean), which are proportional to the wave vertical group velocity
+under linear wave theory (Andrews et al., 1987). Statistically extreme heat 
+flux events thus represent extremes in wave propagation with vertically deep
+planetary wave structures (Dunn-Sigouin and Shaw, 2015); extraordinarily high 
+heat fluxes weaken and warm the polar vortex, whereas negative heat fluxes are 
+generally associated with wave reflection that can dynamically cool and 
+strengthen the vortex. 
+
+Mismatches between modeled and observed planetary wave characteristics can
+be tied to issues/biases related to their sources in the troposphere 
+(Shaw et al., 2014b, England et al., 2016), as well as biases in the 
+stratospheric circulation that affect how such waves propagate. These 
```

Comment:
Can you expand on this a bit more for a non-expert audience of general model developers?